anycloud/node/src/datastore.ts

```diff
@@ -30,65 +34,94 @@ export class DataStore {
     });
     return await response.text();
   }
+
   async get(dsKey: string): Promise<string> {
```

Comment:
This return type is still wrong, since you `return undefined` in some paths. It should be `Promise<string | undefined>`